tests/test_kernelbuilder.py

```diff
@@ -249,6 +250,7 @@ def test_redhat_config_glob_failure(self, mock_check_call, mock_glob,
         # pylint: disable=W0212,E1101
         mock_check_call.return_value = ''
         mock_glob.return_value = []
+        self.kbuilder.rh_configs_glob = "redhat/configs/kernel-*-x86_64.config"
```

Comment:
Why are we adding these?

Reply:
This was my idea. The tests use a minimal configuration of kbuilder which doesn't have the glob set, but it tries to test the `__make_redhat_config` too. If the glob is not set, the method tries to format `None` into the config path. The tests were working in the previous versions by sheer luck that `format` retypes the `None` object to string, but our new method doesn't retype things so it broke the tests (which shouldn't have been working in the first place, since we shouldn't be testing a method without setting the needed arguments).

So this addition ensures the `__make_redhat_config` is being tested properly, and fixes the travis failure since the test issue was discovered by this change.